docs/design/operator-bundle.md

```diff
@@ -44,18 +62,7 @@ annotations:
 *Notes:*
 * In case of a mismatch, the `annotations.yaml` file is authoritative because the on-cluster operator-registry that relies on these annotations has access to the yaml file only.
 * The potential use case for the `LABELS` is - an external off-cluster tool can inspect the image to check the type of a given bundle image without downloading the content.
-
-This example uses [Operator Registry Manifests](https://github.com/operator-framework/operator-registry#manifest-format) format to build an operator bundle image. The source directory of an operator registry bundle has the following layout.
-```
-$ tree test
-test
-├── testbackup.crd.yaml
-├── testcluster.crd.yaml
-├── testoperator.v0.1.0.clusterserviceversion.yaml
-├── testrestore.crd.yaml
-└── metadata
-    └── annotations.yaml
-```
+* Currently, the annotations for bundle manifests and metadata are `manifests/` and `metadata/` and those values are immutable as they are constants.
```

Comment:
Can you clarify this statement? As a user, I can change the metadata/manifests paths in the `annotations.yaml` that I put into my bundle image, so why shouldn't I?

My understanding is that the registry server currently won't respect the value of either annotation (instead using the hardcoded paths you mentioned). If that's right, let's call it out explicitly.

Reply:
If users want to customize annotations via their own Dockerfiler and annotations, they are free to do so. This statement is meant for folks who use the bundle library to generate, build and validate the bundle image. I'm happy to adjust the sentence to make it a bit clearer. Do you have a suggestion in mind?